### ISSUE TITLE
Why do you use different weight_decay ?

### ISSUE THREAD

**Issue by fabiozappo** [microsoft/Swin-Transformer#176]
In this code snippet ([link](https://github.com/microsoft/Swin-Transformer/blob/main/optimizer.py#:~:text=def%20set_weight_decay(model,no_decay%2C%20%27weight_decay%27%3A%200.%7D%5D) you set some layers weight decay to zero. Is there any reason why you do this? Does it has a high impact on training results?

Btw thank you for your work :)

**Comment by fanrz** [comment]
hope for your test result

**Comment by FiorenzoParascandolo1** [comment]
Bias term and scale factor in LayerNorm should be excluded from weight decay
